Sources/Template.swift

```diff
@@ -68,3 +68,8 @@ public struct Template {
         return Renderer(ast: ast, partials: partials.mapValues { $0.ast }).render(with: context)
     }
 }
+
+extension Template: Equatable {}
+public func ==(lhs: Template, rhs: Template) -> Bool {
```

Comment:
this now can be implemented as `public static func` inside the extension. just so you know! (: